product_docs/docs/pem/9/monitoring_performance/notifications.mdx

```diff
@@ -265,3 +265,33 @@ You can use the following command to confirm that Nagios is properly configured:
 After confirming that Nagios is configured correctly, restart the Nagios service:
 
 `/usr/local/nagios/bin/nagios -d /usr/local/nagios/etc/nagios.cfg`
+
+## Customizing email templates
+
+Postgres Enterprise Manager monitors your system for conditions that require user attention and sends email notifications. Use the `Email Templates` to customize the email subject and the payload that the server sends if the current values deviate from the threshold values specified by an alert.
+
```

Comment:
Not sure why Email Templates is capitalized and tagged as code. Seems like it should just be "email templates," no tagging?

Reply:
This is how it is on UI

Reply:
Changed it to **Email Templates**  to match it with same kind reference across the doc